arm-sql/2014-04-01/swagger/capabilities.json

```diff
@@ -118,7 +119,9 @@
                 "id": {
                     "readOnly": true,
                     "type": "string",
-                    "description": "The unique ID of the service objective."
+                    "format": "uuid",
```

Comment:
autorest linter tags this as

`WARNING: Guid used at the #/Definitions/ServiceObjectiveCapability/.../id. Usage of Guid is not recommanded. If GUIDs are absolutely required in your service, please get sign off from the Azure API review board.`

@ravbhatnagar : what's the recommended change here? I do not have enough context on why guids are bad and what's the right way to fix this. I'll appreciate it if you can fill me and Jared in here.